sjasm/lua_sjasm.cpp

```diff
@@ -406,6 +410,7 @@ static void lua_impl_init() {
 			.addFunction("get_byte", lua_sj_get_byte)
 			.addFunction("get_word", lua_sj_get_word)
 			.addFunction("get_device", GetDeviceName)		// no error/warning, can be called directly
+			.addFunction("get_module_namespace", lua_sj_get_module)
```

Comment:
```suggestion
			.addFunction("get_module", lua_sj_get_module)
```
I think this would suffice, at least it makes it match the end part of implementation function name.

Reply:
My first test was only sj.get_module(), but it returns the full namespace with nested modules.

Reply:
hm.. still feels overly long/verbose compared to other directives and lua bindings, for me even `get_modules` would suffice, but considering how often this will be used (very rarely, right?), I'm starting to lean toward your original proposal.

But then maybe refactor implementation function to be also `lua_sj_get_module_namespace`, so they match in suffix (IIRC most of them does match, so it will keep it somewhat "in style") and make it easier to "find all" across sources.

Reply:
ok, i changed the function to sj.get_modules() - we have a consensus :)